src/pages/Home/components/Filter/types.ts

```diff
@@ -30,4 +31,7 @@ export interface IFilterProps {
   data: IFilterFormProps;
   open: true;

```

Comment:
Shouldn't this be set to boolean instead?

Reply:
yes, well noted